apps/css-workshop/src/pages/table.astro

```diff
@@ -2261,41 +2261,77 @@ import Checkbox_ from '../components/Checkbox.astro';
 
           <span class='iui-table-paginator-pages-group'>
             <button
```

Comment:
Should these also be `Button_`?

Reply:
Done in `table.astro` and `breadcrumbs.astro`.